src/utils/manageAriaHidden.js

```diff
@@ -1,30 +1,30 @@
-const BLACKLIST = ['template', 'script', 'style']
+const BLACKLIST = ['template', 'script', 'style'];
```

Comment:
Most of these changes are just Prettier - apparently now you don't use `semi: false`?

Might behoove you to do a PR where you reformat the whole codebase to match current rules so that PR diffs are more discernible?